### DISCUSSION TITLE
User specific content and header not loading

### DISCUSSION THREAD

**Discussion by Hyperscale1** [community/community#50460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Github won't load the homepage or any user details for the header even when logging in and out again.

Refference: https://cdn.discordapp.com/attachments/949907859011956756/1087172656597581834/image.png

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Hyperscale1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
